applications/StructuralMechanicsApplication/custom_conditions/moving_load_condition.cpp

```diff
@@ -0,0 +1,270 @@
+template< std::size_t TDim, std::size_t TNumNodes >
+Condition::Pointer MovingLoadCondition< TDim, TNumNodes>::Create(IndexType NewId,GeometryType::Pointer pGeom,PropertiesType::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<MovingLoadCondition>(NewId, pGeom, pProperties);
+}
+
+//************************************************************************************
+//************************************************************************************
+template< std::size_t TDim, std::size_t TNumNodes >
+Condition::Pointer MovingLoadCondition< TDim, TNumNodes>::Create( IndexType NewId, NodesArrayType const& ThisNodes,  PropertiesType::Pointer pProperties ) const
+{
+    return Kratos::make_intrusive<MovingLoadCondition>( NewId, GetGeometry().Create( ThisNodes ), pProperties );
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+template< std::size_t TDim, std::size_t TNumNodes >
+Condition::Pointer MovingLoadCondition< TDim, TNumNodes>::Clone (
+    IndexType NewId,
+    NodesArrayType const& ThisNodes
+    ) const
+{
+    KRATOS_TRY
+
+    Condition::Pointer p_new_cond = Kratos::make_intrusive<MovingLoadCondition>(NewId, GetGeometry().Create(ThisNodes), pGetProperties());
+    p_new_cond->SetData(this->GetData());
+    p_new_cond->Set(Flags(*this));
+    return p_new_cond;
+
+    KRATOS_CATCH("");
+}
+
+//******************************* DESTRUCTOR *****************************************
+//************************************************************************************
+template< std::size_t TDim, std::size_t TNumNodes >
+MovingLoadCondition< TDim, TNumNodes>::~MovingLoadCondition()
+{
+}
+
+//************************************************************************************
+//************************************************************************************
+template< std::size_t TDim, std::size_t TNumNodes >
+void MovingLoadCondition< TDim, TNumNodes>::CalculateAll(
+    MatrixType& rLeftHandSideMatrix, VectorType& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo,
+    const bool CalculateStiffnessMatrixFlag,
+    const bool CalculateResidualVectorFlag
+    )
+{
+    KRATOS_TRY
+
+    const unsigned int NumberOfNodes = GetGeometry().size();
+    const unsigned int Dimension = GetGeometry().WorkingSpaceDimension();
+
+    const unsigned int block_size = this->GetBlockSize();
+    // Resizing as needed the LHS
+    unsigned int MatSize = NumberOfNodes * block_size;
+   /* for (const auto& r_node : GetGeometry().Points()) {
+        if (r_node.HasDofFor(ROTATION_X)) { MatSize += 1; }
+        if (r_node.HasDofFor(ROTATION_Y)) { MatSize += 1; }
+        if (r_node.HasDofFor(ROTATION_Z)) { MatSize += 1; }
+    }*/
+
+
+    if ( CalculateStiffnessMatrixFlag == true ) //calculation of the matrix is required
```

Comment:
The style is 
if (...) {
} else {
}

Reply:
Just mentioning that we've never positioned about this. The musts are collected in the [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).